src/pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import Firebase from "../../util/Firebase"
+import "./LoginPage.scss"
+import "../../App.scss"
+import UserContext from "../../context/UserContext"
+
+
+class LoginPage extends React.Component {
+
+    static contextType = UserContext;
+
+    state = {
+        email: "",
+        password: "",
+    };
+
+    handleLogin = async (event) => {
+        event.preventDefault();
+        const { email, password } = this.state;
+        try {
+            const user = await Firebase.loginUser(email, password);
+            this.context.onLogin(user); // update user info at the top level
+        } catch (err) {
+            console.log(err);
+        }
+    };
+
+    handleChange = (event) => {
+        event.preventDefault();
+        this.setState({ [event.target.name]: event.target.value });
```

Comment:
what are the array brackets for? I get how the is updating email/password, but the brackets confuse me

Reply:
I believe event.target.name basically acts as a key in a key-value pair, so it needs to be bracket-enclosed.

Reply:
If it's a key would you need to put the brackets next to the pair you are indexing though, like `whateverPairName[event.target.name]`? Lone brackets to me insinuate creating an array

Reply:
If you can test this and show me its working I'll chill just don't trust inherently sorry g

Reply:
![Screenshot 2023-01-03 at 5 16 59 AM](https://user-images.githubusercontent.com/86818322/210355741-7dcf6adc-6717-4eb4-bf57-a709a0e42e76.png)

right above Firebase error - logging state to console right before it's used by Firebase.loginUser, email and password values are being assigned correctly

Reply:
you right